### PR TITLE
Fixes apptainer#62 remove references to commercial remote `keyserver`

### PR DESCRIPTION
**Pull request by PaulCharlton**
Fixes apptainer#62 remove references to commercial remote `keyserver`

### CODE REVIEW

File: internal/pkg/keymanager/error.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+// software.
+
+package keymanager
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	jsonresp "github.com/sylabs/json-resp"
```
Comment (DrDaveD) [inline internal/pkg/keymanager/error.go:16]: Did you intend to still rely on a sylabs module?

Comment (PaulCharlton) [reply internal/pkg/keymanager/error.go:16]: @DrDaveD yes - that needs to be a separate PR -- been doing one PR per module reference

Comment (DrDaveD) [reply internal/pkg/keymanager/error.go:16]: Ok

**Review by DrDaveD — CHANGES_REQUESTED** [review]
Needs to be rebased after #72 is merged because those changes are included here.

**Comment by PaulCharlton** [comment]
@DrDaveD @ikaneshiro removed dependency on `dead code removal` commit; fixed merge conflict with golang 1.15 patch removal

**Comment by PaulCharlton** [comment]
@ikaneshiro @DrDaveD this appears to be ready - green -- and no further dependencies on other PR

**Pull request closed without merging**